src/main/resources/application.yml

```diff
@@ -125,6 +128,9 @@ nakadi:
       commitTimeout: 5 # seconds
   subscription:
     maxPartitions: 30
+kpi:
+  config:
+    stream-data-collection-frequency-ms: 100
```

Comment:
Why do you specify a different value for the test environment? As I see you don't have a test for consumption part. BTW, do you want to have a test for that? :)

Reply:
I tried to write the test. I stopped after I spent 2 hours. It's impossible to unit test the subscription API. I thought about writing an acceptance. I think I could do that =)